extension/chrome/settings/modules/add_key.htm

```diff
@@ -49,27 +49,26 @@ <h1>Add Private Key <span id="spinner_container"></span></h1>
           <input id="input_passphrase" class="input_passphrase" type="text" placeholder="Private Key Passphrase" />
         </div>
         <div class="line">
-          <div class="squaredFour">
-            <input type="checkbox" value="" id="e_rememberPassphrase" name="check"
-              class="input_passphrase_save squaredFour" checked="">
-            <label class="checkboxSquaredOne" for="e_rememberPassphrase"></label>
-          </div>
-          <span class="labelCopy">Remember Pass Phrase after closing browser.</span>
+          <label>
```

Comment:
We don't seem to have any test for this particular choice.

We have a test for changing this later in settings but not during setup. I'll make a new issue for this.